Makefile

```diff
@@ -108,15 +111,40 @@ tag: ## Set a docker tag to the image.
 push: ## Push docker image.
 	docker push $(IMAGE_PREFIX)pvc-autoresizer:$(IMAGE_TAG)
 
+##@ Chart Testing
+
+.PHONY: ct-lint
+ct-lint: ## Lint and validate a chart.
+	docker run \
+		--rm \
+		--user $(shell id -u $(USER)) \
+		--workdir=/data \
+		--volume $(shell pwd):/data \
+		quay.io/helmpack/chart-testing:v$(CHART_TESTING_VERSION) \
+		ct lint --config ct.yaml
+
+.PHONY: ct-install
+ct-install: ## Install and test a chart.
+	docker run \
+		--rm \
+		--user $(shell id -u $(USER)) \
+		--network host \
+		--workdir=/data \
+		--env KUBECONFIG=/kubeconfig \
+		--volume ~/.kube/config:/kubeconfig:ro \
+		--volume $(shell pwd):/data \
+		quay.io/helmpack/chart-testing:v$(CHART_TESTING_VERSION) \
+		ct install --config ct.yaml
+
 ##@ Tools
 
 .PHONY: tools
 tools: staticcheck setup-envtest
 
 .PHONY: staticcheck
 staticcheck: ## Install staticcheck
-	if ! which staticcheck >/dev/null; then \
-		env GOFLAGS= go install honnef.co/go/tools/cmd/staticcheck@latest; \
+	if [ ! -e $(STATICCHECK) ] || [ "$$($(STATICCHECK) --version | awk '{print $$3}')" != "(v$(STATICCHECK_VERSION))" ]; then \
```

Comment:
As I said, we can use latest statickcheck. (https://github.com/topolvm/pvc-autoresizer/pull/169#discussion_r1101146060)
Could you revert it?

Reply:
@toshipp Sorry, I fixed it.